elodie/geolocation.py

```diff
@@ -146,8 +146,6 @@ def place_name(lat, lon):
                 #  set the most specific as the default.
                 if('default' not in lookup_place_name):
                     lookup_place_name['default'] = address[loc]
-
-    if(lookup_place_name is not {}):
```

Comment:
yes, it did. But I thought the if statement was excessive, because I thought this if statement would cover if(geolocation_info is not None and 'address' in geolocation_info): that scenario.
Because lookup will return None if there is an issue with network connectivity. But I may be wrong,